src/core/login.ts

```diff
@@ -35,6 +35,11 @@ export class Login {
     });
   }
 
+  async userCookie(JSESSIONID: string, li_at: string): Promise<Client> {
```

Comment:
Please pass the arguments as an Object. I prefer to be consistent on that matter.